src/System.Private.CoreLib/shared/System/IO/Stream.cs

```diff
@@ -189,6 +189,73 @@ private int GetCopyBufferSize()
             return bufferSize;
         }
 
+        public virtual void CopyTo(ReadOnlySpanAction<byte, object> callback, object state, int bufferSize)
```

Comment:
state should be nullable, same for the objects used as generic type parameters.

Reply:
Fixed => API changed